README.md

```diff
@@ -123,4 +123,31 @@ We re-expose this component from [@solid/react](https://github.com/solid/react-c
   <LogoutButton/>
 ```
 
+### Uploader
+This component allows you to upload files in your POD. This is using React render prop patterns.
+
+The Uploader component contains the file uploading logic and Solid integration, but does not include the UI. You may pass in your own UI as demonstrated here:
+
+```javascript
+<Uploader
+  {...{
+    fileBase: "https://example.org/public",
+    render: (props) => (
+      <ProfileUploader {...{ ...props }} />
+    )
+  }}
+/>
+```
+
+We have a basic ProfileUploader component in demo to show you how you can add your own Uploader UI.
```

Comment:
This sentence needs cleaning up. What is it trying to convey? Which demo does this refer to?

Reply:
Basic UI example to show you how can you works with Uploader component.

Reply:
Referring to the code below?